openwisp_controller/config/migrations/0015_add_relevant_permissions_to_admin_and_operators.py

```diff
@@ -0,0 +1,64 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+from django.contrib.auth.management import create_permissions
+
+
+def create_default_permissions(apps, schema_editor):
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True
+        create_permissions(app_config, apps=apps, verbosity=0)
+        app_config.models_module = None
+
+
+def assign_permissions_to_groups(apps, schema_editor):
+    Group = apps.get_model('openwisp_users', 'Group')
+    admin = Group.objects.get(name='Administrator')
+    operator = Group.objects.get(name='Operator')
+    operators_and_admins_can_change = ['device', 'config', 'template']
+    operators_read_only_admins_manage = ['vpn']
+    manage_operations = ['add', 'change', 'delete']
+
+    for modelClass in operators_and_admins_can_change:

```

Comment:
rename `modelClass` to `model_name`, don't use camel case notation for variables, camle case is reserved for classes.

Please apply this change to all other instances of `modelClass` in your code (or any other inappropiate use of camel case if present).